ci/script.sh

```diff
@@ -5,5 +5,7 @@ if [[ "$TRAVIS_OS_NAME" = "linux" ]]; then
     sudo docker run -it electron-forge-ci ci/docker.sh $NODE_INSTALLER
 else
     if [[ "$NODE_INSTALLER" = "yarn" ]]; then npm i -g yarn; fi
-    npm run test -- --installer=$NODE_INSTALLER
+    npm run test-coverage -- --installer=$NODE_INSTALLER
+    # Only publish coverage on the macOS yarn agent
+    if [[ "$NODE_INSTALLER" = "yarn" ]]; then cat coverage/lcov.info | node node_modules/coveralls/bin/coveralls.js; fi
```

Comment:
Code coverage is outputted even on failed test runs so we want to publish it anyway I think?  Or do we want to ignore failed runs?